lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -935,6 +935,12 @@ class ConvertAtenSqueezeOp : public OpConversionPattern<AtenOpT> {
     auto newOutputTy = RankedTensorType::get(
         makeShapeLLVMCompatible(newOutputShape), resultElemTy);
 
+    if (newOutputTy.getNumDynamicDims() > 1) {
```

Comment:
`tosa.reshape` can handle only one dimension to be dynamic, not more than one. It doesn't matter what that dimension is. Think about the reshape in MATLAB. This is because if we only one dimension is unknown/varsize it can be inferred based on the total size of the data and the rest of the known sizes.